Dockerfile.latest

```diff
@@ -1,6 +1,9 @@
 FROM opensuse:tumbleweed
 RUN zypper ar -f http://download.opensuse.org/repositories/YaST:/Head/openSUSE_Tumbleweed/ yast
 
+# "zypper dup" synchronizes with the current Tumbleweed (even downgrades if needed)
+RUN zypper --gpg-auto-import-keys --non-interactive dup --no-recommends
```

Comment:
The subsequent `zypper in` calls are followed by `&& zypper clean -a` so I think this one should have it too.